src/main/resources/com/welovecoding/nbeditorconfig/EditorConfigExample.editorconfig

```diff
@@ -0,0 +1,33 @@
+# EditorConfig is awesome: http://EditorConfig.org
```

Comment:
Yepp! We should keep it! My comment was about removing the file that we had before you commit your file. :)